### PR TITLE
Added docs about running a subset of the tests with -- -g <substring>

### PR DESCRIPTION
**Pull request by ElijahCaine**
This will save us countless seconds going forward. @osuosl/devs

### CODE REVIEW

**Comment by tschuy** [comment]
+1

File: docs/source/dev/development.rst [hunk 1]
```diff
@@ -77,6 +77,16 @@ To only test the application, use the test command::
 
     npm test
 
+To only run tests which have a description that match specific regular
+expression run::
+
+    npm test -- -g POST
+
+This will only run tests which have the are in a ``describe('.*POST.*',
```
Comment (matthewrsj) [inline docs/source/dev/development.rst]: wording

File: docs/source/dev/development.rst [hunk 2]
```diff
@@ -77,6 +77,16 @@ To only test the application, use the test command::
 
     npm test
 
+To only run tests which have a description that match specific regular
```
Comment (Kennric) [inline docs/source/dev/development.rst]: That is kind of difficult wording. Maybe "You can select which tests to run using a regular expression..."

**Comment by matthewrsj** [comment]
+1, wording looks much better

File: docs/source/dev/development.rst [hunk 3]
```diff
@@ -77,6 +77,15 @@ To only test the application, use the test command::
 
     npm test
 
+You can select which subset of tests to run using pattern matching::
+
+    npm test -- -g POST
+
+This will only run tests which are within a ``describe('.*POST.*', function()
+{`` block (where ``.*POST.*`` is a regular expression. Essentially the ``-g``
```
Comment (tschuy) [inline docs/source/dev/development.rst]: Also matches ``it`` statements.

**Pull request merged**